### PR TITLE
TokenLoader improvements

### PR DESCRIPTION
**Pull request by matzew**
* updating serialID right up from, to it's value is correct in all blocks
* as well added some loggers

### CODE REVIEW

**Comment by matzew** [comment]
@lfryc @secondsun mind taking a 👀  ❓

File: push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java [hunk 1]
```diff
@@ -134,32 +134,37 @@ public void loadAndQueueTokenBatch(@Observes @Dequeue MessageHolderWithVariants
                 String lastTokenInBatch = null;
                 int tokensLoaded = 0;
                 for (int batchNumber = 0; batchNumber < configuration.batchesToLoad(); batchNumber++) {
+
+                    // increasing the serial ID,
+                    // to make sure it's properly read from all block
+                    ++serialId;
```
Comment (secondsun) [inline push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java]: Is it correct for serialId to be incremented if tokens.size() == 0?

Comment (matzew) [inline push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java]: yes, it's an increment, based on the batch.

Currently the increase only happens here:
* https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java#L144

With the consequence that in other blocks the value is different, like: 
* https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java#L147
* or here, were we added the new log -> https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java#L159

Comment (matzew) [inline push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java]: Would be nice to get @lfryc take a quick look here too, but I think the current way of increasing is wrong

**Comment by matzew** [comment]
@secondsun it's also covered in tests, the `serialId`:
* https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/push/sender/src/test/java/org/jboss/aerogear/unifiedpush/message/jms/TestTokenBatchDeduplication.java
* https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/push/sender/src/test/java/org/jboss/aerogear/unifiedpush/message/jms/TestTokenLoaderTransactionFailForGCM.java#L133

merging ... ⏫

**Pull request merged**